docs/dev/running/aziot-certd.md

```diff
@@ -205,9 +205,13 @@ With this basic file, fill it out depending on what workflow you want to test:
     As mentioned at the beginning, if your config file is not saved at `/etc/aziot/certd/config.toml`, set the `AZIOT_CERTD_CONFIG` env var to its actual path.
 
     ```sh
+    export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$PWD/target/x86_64-unknown-linux-gnu/debug"
+
+    export AZIOT_LOG=aziot=debug
+
     export AZIOT_CERTD_CONFIG='...'
 
-    cargo run -p aziot-certd
+    (exec -a aziot-certd target/x86_64-unknown-linux-gnu/debug/aziotd)
```

Comment:
Does `cargo run -p aziotd -- aziot-certd` not work?

Reply:
`cargo run` passes those as `std::process::Command::arg`s, ie `argv[1..]`. Overriding `argv[0]` is an OS-specific thing in the first place, and `std::process::Command` only supports it for unix.

Reply:
oh, I see, took me a second to remember that `argv[0]` is (traditionally) the executable's name. I now see why this bit of rigmarole is required.

Nonetheless, given that (afaik) we aren't passing any other parameters over the CLI, could we have the code _also_ check for `argv[1]` as well?

Reply:
Yeah, I can add it as a fallback if `argv[0]` is `aziotd`

Reply:
Done.